### PR TITLE
Update the readme to account for 0.0.4 and 0.0.5

### PR DESCRIPTION
**Pull request by cgruber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +209,95 @@ but this can be changed (per resolver instance) like so:
...
+> Environment variables are used in the default configuration, as there is often a need to have
+> a different context in CI without altering code, and maven resolution is often required in build
+> systems used in CI
+
+To use a proxy in the default setup, set the a variable in the environment in which the app you
+wish to use will run with the url of the proxy service (including, optionally, a
+username/password). e.g.:
+
+```sh
+setenv HTTP_PROXY=localhost:8080
```
Comment (rogerhu) [inline README.md]: shouldn't this be export HTTP_PROXY=...?

File: README.md [hunk 2]
```diff
@@ -117,6 +209,95 @@ but this can be changed (per resolver instance) like so:
...
+
+Assuming `artifact_resolver_cli` is a maven-archeologist client, it will route fetches through
+the proxy.
+
+Because some environments need to disallow certain addresses from being routed through the
+proxy (for security or performance reasons), setting a list of url matching infixes into
+the environment variable `NO_PROXY` will allow maven-archeologist to route prefix-matching 
+
+```sh
+setenv HTTP_PROXY=localhost:8080
```
Comment (rogerhu) [inline README.md]: same here?

**Review by rogerhu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cgruber) [reply README.md]: Probably.

File: README.md (hunk 2 above)
Comment (cgruber) [reply README.md]: Yeah. I think you're right.

**Pull request merged**